multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperUtil.java

```diff
@@ -65,8 +67,13 @@ public static String getRef(final JsonNode schema, final String prefix, final St
     return getPojoName(getRefClass(schema), prefix, suffix);
   }
 
+  public static String getLongRefClass(final JsonNode schema) {
```

Comment:
I think we should refactor this method into a single one, try to rethink if its possible by the given argument, if for any reason not posible, please let me know

Reply:
I don't understand

Reply:
What I mean is that you are using practically the same code in both `getLongRefClass` and `getRefClass` methods, I think it could be refactored in only one, to avoid having boilerplate. But as I said previously, if for any reason not posible refactor it, let me know

Reply:
Talking with the owner of the PR its based on a functionality so, for me its ok after the talk